include/zim/archive.h

```diff
@@ -152,7 +152,19 @@ namespace zim
     public:
       template<EntryOrder order> class EntryRange;
       template<EntryOrder order> class iterator;
-      static const OpenConfig DEFAULT_OPEN_CONFIG;
```

Comment:
Wouldn't a `LIBZIM_API` help here?

Reply:
No. Static variable are simply not exported in dll.
See https://stackoverflow.com/questions/2479784/exporting-static-data-in-a-dll